code/rust-sokoban-c02-02/src/main.rs

```diff
@@ -10,6 +10,7 @@ use specs::{
     Write, WriteStorage,
 };
 
+use std::collections::VecDeque;
```

Comment:
so the way the book works is it uses the line numbers in the md files to decide what code snippets to show, if we are adding a new line here we'll have to fix the md files which point to this code file and tweak the line numbers. it's a bit annoying but unfortunately this is how it works.